ledger-core/v2/insolar/contract/virtual.go

```diff
@@ -79,3 +79,27 @@ const (
 func (f InterferenceFlag) IsZero() bool {
 	return f == 0
 }
+
+type SendResultFullFlag byte
```

Comment:
unused, the one that are used are in payload

Reply:
![image](https://user-images.githubusercontent.com/5444090/83020725-957c1480-a031-11ea-9da9-f442caaea835.png)
i named that one this way, it is unused now but will be used in future